### ISSUE TITLE
Add flags options for attachments

### ISSUE THREAD

**Issue by Gioyik** [bzdeck/bzdeck#241]

**Comment by kyoshino** [comment]
Thanks for filing a new issue :smiley_cat: Yes, attachment flags should be edited, especially for r+

**Comment by kyoshino** [comment]
I guess this is a dup of #92.

**Comment by kyoshino** [comment]
This is blocked by [Bug 1251064 - Retrieving products from /rest/product takes too long](https://bugzilla.mozilla.org/show_bug.cgi?id=1251064)
